src/projection/node/create-projection-node.ts

```diff
@@ -1161,6 +1162,27 @@ export function createProjectionNode<I>({
             this.pendingAnimation = sync.update(() => {
                 globalProjectionState.hasAnimatedSinceResize = true
 
+                /**
+                 * Check if this component or any of its ancestors are animating
+                 */
+                if (
+                    process.env.NODE_ENV !== "production" &&
```

Comment:
Smart! But wouldn't this mean that it does not work inside of production builds of FramerStudio?

Reply:
Yes, to cut out the overhead. We don't throw any warnings in prod